model/host/volume.go

```diff
@@ -43,6 +44,14 @@ func UnsetVolumeHost(id string) error {
 		bson.M{"$unset": bson.M{VolumeHostKey: true}}))
 }
 
+func (v *Volume) SetDisplayName(displayName string) error {
+	v.DisplayName = displayName
+	return db.Update(VolumesCollection,
```

Comment:
I'd wrap this. 

also I've been writing routes recently where I only update the local copy if the modification succeeds, which feels better, even though it might not matter much.